astester.imports.js

```diff
@@ -0,0 +1,39 @@
+export default function (memory) {
+  const Storage = new Map();
+  return {
+    env: {
+      memory,
+      abort(msg, file, line, col) {
+      getString(msg)
+
+       console.log("error", text)
+      },
+      log(ptr) {
+       console.log("log", text)
```

Comment:
How can that work ? What is this magic ?

Reply:
yes indeed it was ! issue solve with getString